backend/modules/publishing/router.py

```
@@ -1 +1,375 @@
+    except Exception as e:
+        logger.error(f"Error fetching draft simulation {simulation_id}: {e}")
+        raise HTTPException(
+            status_code=500,
+            detail=f"Failed to fetch draft simulation: {str(e)}"
+        )
+
+
+@router.post("/publish/{simulation_id}", response_model=PublishResponse)
+async def publish_simulation(
+    simulation_id: int,
+    publish_request: SimulationPublishRequest,
```

Ruff (0.14.7)

Unused function argument: `publish_request`

(ARG001)